jbi/router.py

```diff
@@ -37,8 +37,11 @@ def root(request: Request, settings: Settings = Depends(get_settings)):
 @router.head("/__heartbeat__")
 def heartbeat(response: Response, actions: Actions = Depends(get_actions)):
     """Return status of backing services, as required by Dockerflow."""
-    health_map = jbi_service_health_map(actions)
-    health_checks = []
+    health_map = {
+        "bugzilla": bugzilla.check_health(),
+        "jira": jira.check_health(actions),
```

Comment:
So, we only leave `check_health` at the module level. Would it make more sense to have it as a client method to align it with the rest of the actions code? eg. `jira.get_client().check_health(actions)` ?

Reply:
I think this comment kind of goes with `https://github.com/mozilla/jira-bugzilla-integration/pull/224#discussion_r958466469`? Maybe we should have a design discussion about classes / clients/ method access and make a change outside this PR.